packages/cds-lsp/package.yaml

```
@@ -0,0 +1,18 @@
+---
+name: cds-ls
+description: Language server for CDS
+homepage: https://cap.cloud.sap
+licenses:
+  - proprietary
+languages:
+  - CDS
+categories:
+  - LSP
+  # - Formatter
+
+source:
+  id: pkg:npm/@sap/cds-lsp@7.4.0
+
+bin:
+  cds-lsp: npm:cds-lsp --stdio
```

These entries should only provide a target for the executable, so we can't include arguments here:

```suggestion
  cds-lsp: npm:cds-lsp
```

The command to actually run the LSP server should be configured in nvim-lspconfig, for example. Mason only makes the executable available.